com.unity.render-pipelines.universal/Runtime/Passes/ScriptableRenderPass.cs

```diff
@@ -28,7 +28,9 @@ public enum ScriptableRenderPassInput
         BeforeRendering = 0,
         BeforeRenderingShadows = 50,
         AfterRenderingShadows = 100,
-        BeforeRenderingPrepasses = 150,
+        BeforeRenderingPrePasses = 150,
+        [Obsolete("Obsolete, to match the capital from 'Prepass' to 'PrePass'")]
```

Comment:
Also add [EditorBrowsable(EditorBrowsableState.Never)] so it doesn't show up in code completion/assist tooling.

Reply:
I added it